HACKING.rst

```diff
@@ -21,7 +21,8 @@ Then you can run tests with:
 
    make
 
-The default Python version is 2, to run tests against Python 3 just
+The default Python interpreter is `python`,
```

Comment:
This no longer makes sense.

Reply:
It's a not for the Debian Miantainer --> myself

Reply:
There's this thingie too, but it can stay https://sources.debian.org/src/systemfixtures/0.6.7-1/debian/patches/using-python3-in-tests.patch/ ; I don't care about making a "perfect package"

Reply:
YES !